controllers/hvpa_controller_test.go

```diff
@@ -217,11 +218,34 @@ var _ = Describe("#TestReconcile", func() {
 			Expect(c.Status().Update(context.TODO(), &p)).To(Succeed())
 
 			Eventually(func() bool {
-				h := &autoscalingv1alpha1.Hvpa{}
+				h := &hvpav1alpha1.Hvpa{}
 				c.Get(context.TODO(), types.NamespacedName{Name: instance.Name, Namespace: instance.Namespace}, h)
 				return h.Status.OverrideScaleUpStabilization
 			}, timeout).Should(BeTrue())
 
+			// Update VPA status, let HVPA scale
+			hvpa := &hvpav1alpha1.Hvpa{}
+			Expect(c.Get(context.TODO(), types.NamespacedName{Name: instance.Name, Namespace: instance.Namespace}, hvpa)).To(Succeed())
```

Comment:
Can we add some additional test cases to cover the new scenarios because of the new control over HPA scaling?